app/cronjob/trailingTrade/step/remove-last-buy-price.js

```diff
@@ -192,6 +209,7 @@ const execute = async (logger, rawData) => {
       minNotional,
       openOrders
     });
+	
```

Comment:
This is the unnecessary line change.
Are you using Visual Studio Code?
If you use Visual Studio Code, then it will automatically format your code when you save it.
In addition, it will give any formatting error message like below:

![image](https://user-images.githubusercontent.com/5715919/120090366-3752a780-c145-11eb-8fde-b3e3793964f9.png)

Reply:
Yes, I'm using. Do I need to activate something?